<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenArkWindow</class>
 <widget class="QMainWindow" name="OpenArkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">OpenArk</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/openark.qrc">
    <normaloff>:/OpenArk/OpenArk.ico</normaloff>:/OpenArk/OpenArk.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QTabBar::tab{
	min-height:25px;
}
</string>
       </property>
       <property name="currentIndex">
        <number>6</number>
       </property>
       <widget class="QWidget" name="tabProcessMgr">
        <attribute name="title">
         <string>Process</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabKernel">
        <attribute name="title">
         <string>Kernel</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabCoderKit">
        <attribute name="title">
         <string>CoderKit</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabScanner">
        <attribute name="title">
         <string>Scanner</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabBundler">
        <attribute name="title">
         <string>Bundler</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabUtilities">
        <attribute name="title">
         <string>Utilities</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabReverse">
        <attribute name="title">
         <string>Reverse</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QWidget" name="consoleWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="cmdOutWindow">
          <property name="styleSheet">
           <string notr="true">background: #293134;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#21daff;&quot;&gt;OpenArk Console&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#e0e2e4;&quot;&gt;Copyright (C) 2019 BlackINT3 &lt;/span&gt;&lt;a href=&quot;https://github.com/BlackINT3/OpenArk&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#8ccf34;&quot;&gt;https://github.com/BlackINT3/OpenArk&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Fixedsys</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>C:\&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="cmdInput"/>
          </item>
          <item>
           <widget class="QPushButton" name="cmdButton">
            <property name="text">
             <string>Command Help</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionConsole"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionOnTop"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionZhcn"/>
    </widget>
    <addaction name="actionSettings"/>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>&amp;Plugins</string>
    </property>
    <addaction name="actionCoderKit"/>
    <addaction name="separator"/>
    <addaction name="actionScanner"/>
    <addaction name="actionBundler"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionManuals"/>
    <addaction name="separator"/>
    <addaction name="actionGithub"/>
    <addaction name="separator"/>
    <addaction name="actionCheckUpdate"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionOnTop"/>
   <addaction name="separator"/>
   <addaction name="actionGithub"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/console.ico</normaloff>:/OpenArk/console.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Console</string>
   </property>
   <property name="toolTip">
    <string>Open Console Window</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+C</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/open.ico</normaloff>:/OpenArk/open.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Ex&amp;port</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/exit.ico</normaloff>:/OpenArk/exit.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionOutput">
   <property name="text">
    <string>Output</string>
   </property>
  </action>
  <action name="actionHelloCoder">
   <property name="text">
    <string>HelloCoder</string>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionGithub">
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/gitub.ico</normaloff>:/OpenArk/gitub.ico</iconset>
   </property>
   <property name="text">
    <string>Project on Github</string>
   </property>
   <property name="toolTip">
    <string>Project on Github</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/OpenArk.ico</normaloff>:/OpenArk/OpenArk.ico</iconset>
   </property>
   <property name="text">
    <string>About OpenArk</string>
   </property>
  </action>
  <action name="actionCheckUpdate">
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/update.ico</normaloff>:/OpenArk/update.ico</iconset>
   </property>
   <property name="text">
    <string>Check for Updates...</string>
   </property>
  </action>
  <action name="actionGithub_2">
   <property name="text">
    <string>Project on Github</string>
   </property>
  </action>
  <action name="actionManuals">
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/help.ico</normaloff>:/OpenArk/help.ico</iconset>
   </property>
   <property name="text">
    <string>Manuals</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/refresh.ico</normaloff>:/OpenArk/refresh.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/save.ico</normaloff>:/OpenArk/save.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/settings.ico</normaloff>:/OpenArk/settings.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionCoderKit">
   <property name="text">
    <string>CoderKit</string>
   </property>
  </action>
  <action name="actionScanner">
   <property name="text">
    <string>Scanner</string>
   </property>
  </action>
  <action name="actionCleaner">
   <property name="text">
    <string>C&amp;leaner</string>
   </property>
  </action>
  <action name="actionBundler">
   <property name="text">
    <string>Bundler</string>
   </property>
  </action>
  <action name="actionOnTop">
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/ontop.ico</normaloff>:/OpenArk/ontop.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Allways On Top</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset Window</string>
   </property>
  </action>
  <action name="actionModule">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/module.ico</normaloff>:/OpenArk/module.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Module</string>
   </property>
   <property name="toolTip">
    <string>Open Module Window</string>
   </property>
  </action>
  <action name="actionHandle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/handle.ico</normaloff>:/OpenArk/handle.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Handle</string>
   </property>
   <property name="toolTip">
    <string>Open Handle Window</string>
   </property>
  </action>
  <action name="actionMemory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/memory.ico</normaloff>:/OpenArk/memory.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Memory</string>
   </property>
   <property name="toolTip">
    <string>Open Memory Window</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionZhcn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>简体中文</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../res/openark.qrc">
     <normaloff>:/OpenArk/main/run.ico</normaloff>:/OpenArk/main/run.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/openark.qrc"/>
 </resources>
 <connections/>
</ui>
